generators/writing/templates/src/index.ejs

```diff
@@ -31,6 +31,9 @@ server.on('listening', async () => {
   <%- insertFragment('listening') %>
 <%# -%>
 <%# --- if-2 starts below. -%>
+<% if (Object.keys(specs.services).map(n => specs.service[n]).find(s => s.adapter === 'mongodb')) { -%>
```

Comment:
@eddyystop Is there a better way to do this conditional?

Reply:
I've used code like that and can't think of a better way.